lib/shared/config/convert-config.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+var path = require('path');
+var logger = require('fancy-log');
+var chalk = require('chalk');
+
+module.exports = {
+
+  description: function(value, configFile) {
```

Comment:
I got it. I'll remove these validations.

But only one thing, converting the path of `flags.gulpfile` is necessary because users may set a relative path from the config file.